server/game/cards/characters/01/thequeenofthorns.js

```diff
@@ -6,24 +6,17 @@ class TheQueenOfThorns extends DrawCard {
             when: {
                 afterChallenge: (event, challenge) => challenge.winner === this.controller && challenge.isParticipating(this) && challenge.challengeType === 'intrigue'
             },
-            handler: () => {
-                this.game.promptForSelect(this.controller, {
-                    activePromptTitle: 'Select a character to put in play',
-                    source: this,
-                    cardCondition: card => card.location === 'hand' && card.getCost() && card.getType() === 'character' && card.isFaction('tyrell'),
-                    onSelect: (p, card) => this.onCardSelected(p, card)
-                });
+            target: {
+                activePromptTitle: 'Select a character to put into play',
+                cardCondition: card => card.location === 'hand' && card.controller === this.controller &&
```

Comment:
There **may** be something client side that disallows clicking on facedown / hidden / opponent hand cards, but I'm not sure where it happens. We've also recently talked about being able to click opponent's hand cards as an alternate UI for things like His Viper Eyes and other "look at hand" abilities (e.g. make those cards not facedown and then do a normal card select prompt).

I don't think the controller check hurts anything here, but I'm not sure we've been consistent about it either.